internal/aws/kinesis/kinesis.go

```diff
@@ -20,9 +20,9 @@ import (
 )
 
 const (
-	kplMagicLen   = 4  // Length of magic header for KPL Aggregate Record checking.
-	kplDigestSize = 16 // MD5 Message size for protobuf.
-	kplMaxBytes   = 1024 * 1024
+	kplMagicLen   = 4         // Length of magic header for KPL Aggregate Record checking.
+	kplDigestSize = 16        // MD5 Message size for protobuf.
+	kplMaxBytes   = 1000 * 25 // 25KB is the minimum size of a PUT Payload unit.
```

Comment:
docs: the mismatch of variable name vs comment for maximum and minimum is a bit confusing, maybe some comments could clear up how this is used or we could change the name?

Reply:
I think the comment can be improved -- the value is the maximum amount of bytes we want to allow in the aggregated (KPL) record because it is the minimum amount charged by the Kinesis service.